src/configuration.js

```diff
@@ -31,47 +31,8 @@ import LoggingReporter from './reporters/logging_reporter';
 import RemoteSampler from './samplers/remote_sampler';
 import Tracer from './tracer';
 import UDPSender from './reporters/udp_sender';
-import {Validator} from 'jsonschema';
 import opentracing from 'opentracing';
 
-let configSchema = {
-    'type': 'object',
-    'required': ['jaeger'],
-    'properties': {
-        'jaeger': {
-            'type': 'object',
-            '$ref': '/jaeger'
-        }
-    },
-};
-
-let jaegerSchema = {
```

Comment:
why remove this?  My proposal in another review was to keep the schema definition, just not do the validation (only do it in tests).